docs/tutorials/user-transpiled-circuits.ipynb

```diff
@@ -0,0 +1,2511 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.                                    .probabilities_dict()
</pre><p>Is there any reason you want to break this into a second line?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/qiskit-ibm-runtime/pull/231/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>